packages/components/Button/src/FAB/FAB.types.ts

```diff
@@ -1,13 +1,20 @@
 import { ButtonSlotProps, ButtonCoreTokens, ButtonCoreProps } from '../Button.types';
 import { ShadowProps } from '@fluentui-react-native/experimental-shadow';
+import { PressableProps } from 'react-native';
 
 export const fabName = 'FAB';
 
 export interface FABSlotProps extends ButtonSlotProps {
   shadow?: ShadowProps;
+  ripple?: PressableProps;
 }
+
+export interface FABProps extends ButtonCoreProps {
+  showChildren?: boolean;
+}
```

Comment:
Added this to provide the functionality of collapsible FAB where content can be hidden and shown as required, have added a sample on the tester app

Reply:
Could that be a separate PR from adding the ripple? The ripple addition is a fairly big change and it would be easier to review new functionality if one PR is focused on one addition at a time instead of having them be in the same PR.

Reply:
Alright, will separate the addition of showChildren on FAB and the ripple changes on FAB and Button